src/components/navigation/Navigation.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import Button from '@material-ui/core/Button';
+import { withStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import Search from '@material-ui/icons/Search';
+import Add from '@material-ui/icons/Add';
+import './Navigation.css';
+
+
+const styles = theme => ({
+    button: {
+      margin: theme.spacing.unit,
+    },
+    input: {
+      display: 'none',
+    },
+  });
+
+class Navigation extends React.Component {
+
+    render() {
+        const { classes } = this.props;
```

Comment:
`classes` that stuff comes from `withStyles`?

Reply:
fixed